source/docs/articles/sites/deploys.md

```diff
@@ -0,0 +1,45 @@
+---
+title: The Pantheon Deploy Log
+description: Learn how to use the Deploy Log to manage updates, feature releases and more.
+keywords: deploy, deploy log, how to deploy, deploying
+---
+Whether you're working on a new feature or fixing up bugs, the Deploy Log helps you group a batch of commits into a single deployment. Best practice is to keep logical groups of edits together and then summarzie those groups with a single deployment message.
+
+## Deploy to Test
+The Deploys tool on the test environment enables you to deploy code from the Dev environment, while optionally cloning the the database and files from the Live Environment. The code deployment is performed by adding a git tag to the most recent commit on the Dev environment. Cloning the DB and files from the Live environment completely replaces what was in Test. 
+
+On WordPress site dashboards, cloning the content will expose an option to convert URLs from the Live environment's pattern to the Test environment's, including the protocol from HTTPS to HTTP for encrypted live environments.
+
+Drupal site deploys come with the ability to run `update.php` after  code deploys. `hook update n` and any other necessary database updates will be made automatically on the Test environment. 
+![Deploy Log Test Environment](/source/docs/assets/images/deploy-log-test-env.png)
+
+## Deploy to Live
+The Deploys tool in the Live environment deploys only the code from the Test environment. If you cloned the database and files from the Live environment when you deployed to Test, your Live environment will behave exactly as the Test environment did prior to the deploy. Deploys are performed by adding a git tag to the last commit on the Test environment. 
+All sites have the option of clearing all caches after deploying code. 
+Drupal sites have the option of running update.php after deploying code.
+![Deploy Log Live Environment](/source/docs/assets/images/deploy-log-live-env.png)

```

Comment:
Images are incongruent. This is a Drupal site, while the other is a WordPress site.